aviary/models/test_aircraft/aircraft_for_bench_FwFm.csv

```diff
@@ -146,7 +146,7 @@ aircraft:wing:wetted_area,2396.56,ft**2
 mission:constraints:max_mach,0.785,unitless
 mission:design:cruise_altitude,35000,ft
 mission:design:gross_mass,175400.0,lbm
-mission:design:range,3500,NM
```

Comment:
I wouldn't change this mass because multiple test/benches use this file. In your test where you do need 2000 NM, I would just set this variable locally after loading the csv, but before making any other level2 calls. (prob.aviary_inputs holds all of these at that time)